src/rp2040/hardware_structs/include/hardware/structs/dma.h

```diff
@@ -145,6 +145,11 @@ typedef struct {
     io_rw_32 ints1;
 
     _REG_(DMA_TIMER0_OFFSET) // DMA_TIMER0
+    // (Description copied from DMA_TIMER0 applies similarly all array members)
```

Comment:
I'm not sure we need the "Description copied from" part? Would "(DMA_TIMER0 applies similarly all array members)" be sufficient?